internal/sirius/edit_severance_application_test.go

```diff
@@ -10,22 +10,26 @@ import (
 	"testing"
 )
 
+func convertToBool(b bool) *bool {
```

Comment:
Maybe a bit of a clearer name?
```suggestion
func boolPointer(b bool) *bool {
```